sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -544,4 +702,18 @@ protected void finalize() throws Throwable {
     DebugMode.logV(TAG, "OoyalaSkinLayoutController Finalized");
     super.finalize();
   }
+
+  @Override
+  public boolean onKey(View view, int i, KeyEvent keyEvent) {
+    if(keyEvent.getKeyCode() == KeyEvent.KEYCODE_BACK) {
```

Comment:
I feel that this will always be triggered, verify if you need to check if you're in fullscreen mode and/or stereo.

Reply:
I'll merge this PR but please fix this issue in another one. @IvanSakharovskii